direct_chat.py

```
@@ -212,12 +238,29 @@ async def _handle_agent_mode(
+    
+    Performs a preflight validation of req.repo_url and req.repo_ref, ensures the session exists, persists the user's message, creates an agent job and workspace, runs an adapter readiness check, and schedules the agent execution to run in the background.
+    
+    Returns:
+        JSONResponse: 202 response containing `session_id`, `job_id`, `status`, `phase`, and a `message` indicating the agent workflow was queued.
+    
+    Raises:
+        HTTPException: 412 if repository validation fails or the adapter readiness check reports not ready.
+    """
     log.info(f"Agent mode chat from {user.email}: {req.content[:50]}...")
     session_id = req.session_id
     if session_id: history = _session_history(session_id)
```

Ruff (0.15.12)

[error] Multiple statements on one line (colon)

(E701)